templates/default/cert-manager.yml.erb

```diff
@@ -5271,7 +5271,7 @@ spec:
           type: RuntimeDefault
       containers:
         - name: cert-manager-cainjector
-          image: "quay.io/jetstack/cert-manager-cainjector:v1.11.0"
+          image: "<%= @registry_addr %>/jetstack/cert-manager-cainjector:<%= node['kube-hops']['cert-manager']['version'] %>"
```

Comment:
Can you keep the "quay.io" as well? something like `<%= @registry_addr %>/quay.io/jetstack/cert-manager-cainjector`